html/waf-response.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<header><title>Forbidden</title></header>
```

Comment:
Would it be possible to name the HTML file the same as the status code (403.html)? 

I'm thinking that a nice to have might be for apps to be able to override the default HTML files by specifying them as `file(foo.html)` in the tfvars? It would enable the developers to use branded error pages if they wanted to

Reply:
Yep currently the default is to respond with `403` and this response body
We can add in a feature to allow this to be configurable 👍